blog-posts/event-sourcing-serveless/event-sourcing-serveless-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+published: false
+title: 'The best way to do event sourcing in serverless'
+cover_image:
+description: 'Description of the article'
+tags: serverless, event-sourcing, AWS, CQRS
+series:
+canonical_url:
+---
+
+# Introduction
+
+You probably use a bank application in your every day life and you wonder how they work. They rely on event sourcing to store their data and they compute the balance of your account from all credit/debit events stored. Numerous applications need to store a trace of all events to keep and use the history. If you have that need and you're thinking about implementing it in serverless, you are in the right place. Thus I want to share with you, 2 years of learnings that will lead you to have the most stable and the less painful solution you can go with.
+
+## CQRS and event sourcing
+
+CQRS (Command and Query Responsibility Segregation) is a pattern that separates read and write operations for a data store. It's a good thing to grasp a bit what it is to understand better what will be approached in this article, but don't worry it is not required as it is quite straightforward. Still numerous resources already deal with this topic that you can easily look up on the web.
```

Comment:
CQRS has nothing to do with data storage, but everything to do with interface implementation. It ensures model for read and for write are distinct, but at the end of the day, data persistence layer can be the same for both operation. Event-sourcing focus on data persistence model

Reply:
If you say  there is other resources dealing about CQRS, it's always good to give one or two links ;)